Very flaky test: test_experiment_server

Experiment server is failing quite consistently with:
```
=================================== FAILURES ===================================
__________________ test_experiment_server_ensemble_experiment __________________

tmpdir = local('/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pytest-of-runner/pytest-0/test_experiment_server_ensembl0')
source_root = PosixPath('/Users/runner/work/ert/ert')
capsys = <_pytest.capture.CaptureFixture object at 0x139fbba60>

    @pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7 or higher")
    @pytest.mark.integration_test
    @pytest.mark.timeout(20)
    def test_experiment_server_ensemble_experiment(tmpdir, source_root, capsys):
        shutil.copytree(
            os.path.join(source_root, "test-data", "local", "poly_example"),
            os.path.join(str(tmpdir), "poly_example"),
        )
    
        with tmpdir.as_cwd():
            parser = ArgumentParser(prog="test_main")
            parsed = ert_parser(
                parser,
                [
                    ENSEMBLE_EXPERIMENT_MODE,
                    "poly_example/poly.ert",
                    "--port-range",
                    "1024-65535",
                    "--enable-experiment-server",
                    "--realizations",
                    "0-4",
                ],
            )
    
            FeatureToggling.update_from_args(parsed)
    
>           run_cli(parsed)

ert_tests/cli/test_integration_cli.py:264: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/ert_shared/cli/main.py:56: in run_cli
    asyncio.run(
../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/asyncio/runners.py:44: in run
    return loop.run_until_complete(main)
../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/asyncio/base_events.py:603: in run_until_complete
    self.run_forever()
../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/asyncio/base_events.py:570: in run_forever
    self._run_once()
../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/asyncio/base_events.py:1823: in _run_once
    event_list = self._selector.select(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selectors.KqueueSelector object at 0x13a116cd0>
timeout = 0.9940943789999892

    def select(self, timeout=None):
        timeout = None if timeout is None else max(timeout, 0)
        max_ev = len(self._fd_to_key)
        ready = []
        try:
>           kev_list = self._selector.control(None, max_ev, timeout)
E           Failed: Timeout >20.0s

../../../../hostedtoolcache/Python/3.8.13/x64/lib/python3.8/selectors.py:558: Failed
```

--- a/tests/ert_tests/cli/test_integration_cli.py
+++ b/tests/ert_tests/cli/test_integration_cli.py
@@ -238,6 +238,9 @@ def test_ies(tmpdir, source_root):
 @pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python3.7 or higher")
 @pytest.mark.integration_test
 @pytest.mark.timeout(20)
+@pytest.mark.skip(
+    "Experiment server seemingly causes failures with other tests, possiby bad cleanup"
+)
 def test_experiment_server_ensemble_experiment(tmpdir, source_root, capsys):
     shutil.copytree(
         os.path.join(source_root, "test-data", "local", "poly_example"),

